community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md

```diff
@@ -0,0 +1,99 @@
+The `Create-AzStorageTable` also has the ability to re-create an existing table if that is required.
+
+```powershell
+PS> Create-AzStorageTable -ResourceGroupName "stock" -StorageAccountName "admin" -TableName "products"
+# Creating table 'products' in the storage account 'admin'..."
+```
+
+See the [docs](https://scripting.arcus-azure.net/features/powershell/azure-storage) for more information on this function.
+
+## Azure Resource Manager (ARM)
+
+The `Arcus.Scripting.ARM` PowerShell module contains the capability to inject content into a ARM template, controling the formatting of the injected content.
+After importing the module, the `Inject-ArmContent` function is available to you.
+
+The [docs](https://github.com/arcus-azure/arcus.scripting/blob/master/docs/preview/features/powershell/arm.md) has an example of an OpenAPI specifications you'd want to deploy.
+
+## Azure DevOps
+
+Something that coming up several times but was never actually seen as a separate functionality, was setting Azure pipeline variables. This functionality was scattered across scripts and was making the functions itself unnecessary complex. By separating this functionality in a dedicated module only for Azure DevOps related functionality, we have made using these functions a lot more user-friendly.
+
+The `Set-AzDevOpsVariable` takes in the name and the value of the variable. It's as simple as that!
+
+```powershell
+PS> Set-AzDevOpsVariable -Name "Arcus.KeyVault.VaultUri" -Value "https://arcus.azure.vault.com"
+```
+
+See the [docs](https://scripting.arcus-azure.net/features/powershell/azure-devops) for more information on this function.
+
+## What's on the Horizon
+
+This summary of all the features in this first version is only the beginning of a great scripting story.
+Here are some features that are on the horizon, but please don't heasitate to [make a suggestion](https://github.com/arcus-azure/arcus.scripting/issues/new?template=Feature_request.md) and we're happy to help where we can!
+
+* [Azure API Management](https://github.com/arcus-azure/arcus.scripting/issues?q=is%3Aissue+is%3Aopen+label%3Aarea%3Aapi-management)
+  * [Provide script for restoring an Azure API Management service](https://github.com/arcus-azure/arcus.scripting/issues/76)
+  * [Provide script to backup an Azure API Management service](https://github.com/arcus-azure/arcus.scripting/issues/75)
+  * [Provide script to set the subscriptionKey on an Azure API Management service](https://github.com/arcus-azure/arcus.scripting/issues/39)
+* [Azure Key Vault](https://github.com/arcus-azure/arcus.scripting/issues?q=is%3Aissue+is%3Aopen+label%3Aarea%3Akey-vault)
+  * [Provide script for importing certificate as base64-string into Azure Key Vault](https://github.com/arcus-azure/arcus.scripting/issues/71)
+
```

Comment:
```suggestion

Are we missing something? Don't hesitate to [make a suggestion](https://github.com/arcus-azure/arcus.scripting/issues/new?template=Feature_request.md) and we're happy to help where we can!
```

Reason is that it's best to show what we are planning and only then ask for more suggestions; this feels more natural to me.